packages/cashier/src/pages/payment-agent/payment-agent-list/payment-agent-list.tsx

```diff
@@ -13,6 +13,14 @@ import PaymentAgentDisclaimer from '../payment-agent-disclaimer';
 import { useCashierStore } from '../../../stores/useCashierStores';
 import './payment-agent-list.scss';
 
+// TODO: Remove this declaration after changing label to data-label in Tabs component
+declare module 'react' {
+    interface HTMLAttributes<T> extends React.AriaAttributes, React.DOMAttributes<T> {
+        label?: string;
+        hash?: string;
+    }
+}
+
```

Comment:
@hamid-deriv Is it possible to move it to the `Tabs` component itself? 🤔

Reply:
Good Idea. Sorted.